docs/storybook/src/components/ToolbarComposer.stories.tsx

```diff
@@ -2,16 +2,18 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
+import React from "react";
 import type { Meta, StoryObj } from "@storybook/react";
 import { BadgeType } from "@itwin/appui-abstract";
 import {
   CommandItemDef,
+  ToolbarComposer,
   ToolbarHelper,
   ToolbarItemUtilities,
   ToolbarOrientation,
   ToolbarUsage,
   UiFramework,
-} from "@itwin/appui-react";
+} from "@itwin/appui-react/src/appui-react";
```

Comment:
If we keep importing from the barrel the `autodocs` fails to generate controls from component props: https://storybook.js.org/docs/react/writing-docs/autodocs#the-auto-generated-documentation-is-not-showing-up-in-a-monorepo-setup
Additionally, if I only import a component from sources (and other APIs from the barrel) then at runtime I end up with multiple static instances.

Reply:
I'm worried that we fail to see missing issues by doing that, because then only the stuff built by Storybook will get run, to address this in admin-component-react I instead created a local version of the component I'm testing:

```
export const ToolbarComposer = (props: ToolbarComposerProps) => (
  <ImportedToolbarComposer {...props} />
);

meta.excludeStories = [ "ToolbarComposer"];
```

But if you think this will be safe, we can try it.